src/algebra/big_operators/prime.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2021 Stuart Presnell. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Stuart Presnell
+-/
+
+import data.nat.prime
+import data.finset.basic
+import data.finsupp.basic
+import algebra.big_operators.basic
+
+
+/-!
+# Prime numbers
```

Comment:
This title is too general

Reply:
I'm not sure what's a suitable level of generality for a title.  I've changed it to "Products and sums involving prime numbers".

Reply:
Much better. These headings appear without context in google results, and we don't want searches for `"mathlib prime numbers" preferring this page over the page introducing primes

Reply:
Ah, that hadn't occurred to me, thanks